lib/util/git/index.ts

```diff
@@ -551,6 +551,30 @@ export async function isBranchModified(branchName: string): Promise<boolean> {
   return true;
 }
 
+export async function isBranchConflicted(
+  baseBranch: string,
+  branchName: string
+): Promise<boolean> {
+  if (!branchExists(branchName)) {
+    logger.debug(
+      { branchName },
+      'Branch does not exist - cannot check isBranchConflicted'
+    );
+    return false;
+  }
+  try {
+    await syncGit();
+    await git.reset(ResetMode.HARD);
+    await git.checkout(['-B', branchName, 'origin/' + branchName]);
```

Comment:
Why is it needed to checkout the origin branch? Maybe document it for future

Reply:
Oops, it's the result of copy-paste (fixed now)